.travis.yml

```diff
@@ -15,8 +15,8 @@ install:
   - sudo apt-get update
   - sudo apt-get install -y swig
   - pip install -U tox coveralls
-  - sudo wget https://dist.opendnssec.org/source/softhsm-2.3.0.tar.gz
-  - sudo tar -xzf softhsm-2.3.0.tar.gz
+  - wget https://dist.opendnssec.org/source/softhsm-2.3.0.tar.gz
+  - tar -xzf softhsm-2.3.0.tar.gz
   - cd softhsm-2.3.0
   - sudo ./configure
```

Comment:
I don't think now you  need sudo for configure and make,  only for install. You used to need sudo for these because the tar was sudo'd.

Reply:
I tried it without sudo (when I didnt use sudo for wget and tar), but it didn't work. 
There were some erros, regarding Permission Denied!

Reply:
I'm pretty sure you don't need it:
```
FROM debian:stable

RUN apt-get update && apt-get install -y swig python3-pip wget libssl-dev
RUN pip3 install -U tox coveralls

# drop privs
RUN useradd mortal && mkdir /mundaneopolis && chown mortal:mortal /mundaneopolis
USER mortal:mortal
WORKDIR /mundaneopolis

RUN wget https://dist.opendnssec.org/source/softhsm-2.3.0.tar.gz && tar -zxvf softhsm-2.3.0.tar.gz
WORKDIR /mundaneopolis/softhsm-2.3.0/
RUN ./configure --disable-gost && make

# RUN make install # will fail
USER root:root
RUN make install
```

Reply:
Yup, It does not need sudo. I did a clean install, without sudo and it worked.
Apparently, there is a .log file which is modified by the .configure script. Due to my previous installation, root was the owner of the file.